aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -189,18 +225,17 @@ def _expand_dims_to_match_ome(
         # need to correct channel count if this is a RGB image
         n_samples = ome.images[scene_index].pixels.channels[0].samples_per_pixel
         for d in dims:
+            # SizeC can represent RGB (Samples) data rather
+            # than channel data, whether or not this is the case depends
+            # on what the SamplesPerPixel are for the channel
             if d == "C" and n_samples is not None and n_samples > 1:
                 count = len(ome.images[scene_index].pixels.channels)
+            elif d == "S" and n_samples is not None and n_samples > 1:
+                count = n_samples
```

Comment:
Since `n_samples` is determined _before_ the `for` loop, we might simplify this condition by adding something like this:

```suggestion
        n_samples = ome.images[scene_index].pixels.channels[0].samples_per_pixel
        has_multiple_samples = n_samples is not None and n_samples > 1
        for d in dims:
            # SizeC can represent RGB (Samples) data rather
            # than channel data, whether or not this is the case depends
            # on what the SamplesPerPixel are for the channel
            if d == "C" and has_multiple_samples:
                count = len(ome.images[scene_index].pixels.channels)
            elif d == "S" and has_multiple_samples:
                count = n_samples
```